src/components/Loading/LoadingIndicator.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {
+    Animated,
+    Easing,
+    View,
+    StyleSheet,
+} from 'react-native';
+import Svg, {Circle} from 'react-native-svg';
```

Comment:
Could we save some time and use the `ActivityIndicator` for now? I appreciate the work done here so far. But we usually try to keep things simple. But if there is a great case for this we will look at whether a custom component can solve some problem and create a new ticket for it.